Add anchors for easier deep linking

loisaidasam opened the pull request.

Commit:
Add anchors for easier deep linking

alex commented:
You need to update the HTML template file, if you change this file it'll just get overritten immediately.

alex commented:
Or this might be in the code, not the HTML template.

loisaidasam commented:
Doh

Commit:
Add anchors for easier deep linking to HTML builder

alex commented on print-battleground-state-changes:
```diff
@@ -265,7 +265,9 @@ for (state, timestamped_results) in summarized.items():
     print(tabulate([string_summary(summary) for summary in timestamped_results]))
 
-    html_chunks.append("<table class='table table-bordered'>")
+    # TODO: Use a better slug for "North Carolina (15)" (will just be "north" for now)
```
If we just do `split("(")` that should work. Then replace the " " with a dash or something

loisaidasam replied:
Good point, on it ...

Commit:
Better state slugifier

loisaidasam commented:
```python
In [4]: 'Alaska (3)'.split('(')[0].strip().replace(' ', '-').lower()                                                                                         
Out[4]: 'alaska'

In [5]: 'North Carolina (15)'.split('(')[0].strip().replace(' ', '-').lower()                                                                                
Out[5]: 'north-carolina'
```

alex commented:
Thanks!

loisaidasam commented:
Thank you 🙏
